lib/appsignal/phoenix.ex

```diff
@@ -29,14 +29,15 @@ defmodule Appsignal.Phoenix do
           super(conn, opts)
         rescue
           e ->
+            stacktrace = System.stacktrace
             import Appsignal.Phoenix
             case {Appsignal.TransactionRegistry.lookup(self), extract_error_metadata(e, conn, System.stacktrace)} do
```

Comment:
Makes sense to use the `stacktrace` var here as well to prevent calling the `System.stacktrace` function twice.